### PR TITLE
Use Google comment style and fix typos.

### PR DESCRIPTION
**Pull request by antiagainst**

### CODE REVIEW

File: source/opcode.h [hunk 1]
```diff
@@ -31,166 +31,84 @@
...
 
-/// @brief Find the named Opcode in the table
-///
-/// @param[in] table to lookup
-/// @param[in] name name of Opcode to find
-/// @param[out] pEntry returned Opcode table entry
-///
-/// @return result code
+// Finds the named opcode in the given opcode table. On success, returns
+// SPV_SUCCESS and writes a pointer of the table entry into *entry.
```
Comment (dmircevski) [inline source/opcode.h]: Unclear language: "pointer of" should probably be "pointer to", but then `*entry` isn't a pointer -- it's an abstract type `spv_opcode_desc`. :confused:

File: source/opcode.h [hunk 2]
```diff
@@ -31,166 +31,84 @@
...
-///
-/// @param[out] table to lookup
-/// @param[in] opcode value of Opcode to fine
-/// @param[out] pEntry return Opcode table entry
-///
-/// @return result code
+                                      const char* name, spv_opcode_desc* entry);
+
+// Finds the opcode by enumerant in the given opcode table. On success, returns
+// SPV_SUCCESS and writes a pointer of the table entry into *entry.
```
Comment (dmircevski) [inline source/opcode.h]: Same comment as line 46 above.

File: source/opcode.h [hunk 3]
```diff
@@ -31,166 +31,84 @@
...
-int16_t spvOpcodeResultIdIndex(spv_opcode_desc entry);
-
-/// @brief Determine if the Opcode has capability requirements.
-///
-/// This function does not check if @a entry is valid.
-///
-/// @param[in] entry the Opcode entry
-///
-/// @return zero if false, non-zero otherwise
+// Determines if the Opcode has capability requirements. Returns zero if false,
```
Comment (dmircevski) [inline source/opcode.h]: Consider "opcode".

File: source/opcode.h [hunk 4]
```diff
@@ -31,166 +31,84 @@
...
 int32_t spvOpcodeRequiresCapabilities(spv_opcode_desc entry);
 
-/// @brief Copy an instructions word and fix the endianness
-///
-/// @param[in] words the input instruction stream
-/// @param[in] opcode the instructions Opcode
-/// @param[in] wordCount the number of words to copy
-/// @param[in] endian the endianness of the stream
-/// @param[out] pInst the returned instruction
+// Copies an instructions word and fix the endianness to host native. The source
```
Comment (dmircevski) [inline source/opcode.h]: "fixes"

File: source/opcode.h [hunk 5]
```diff
@@ -31,166 +31,84 @@
...
+// instruction's stream/opcode/endianness is in the words/opcode/endian
+// parameter. The word_count parameter specifies the number of words to copy.
+// Writes copied instruction int *inst.
```
Comment (dmircevski) [inline source/opcode.h]: "into"?

File: source/opcode.h [hunk 6]
```diff
@@ -31,166 +31,84 @@
...
 
-/// @brief Deep comparison of type declaration instructions
-///
-/// @param[in] pTypeInst0 type definition zero
-/// @param[in] pTypeInst1 type definition one
-///
-/// @return zero if false, non-zero otherwise
+// Deep equal comparison of type declaration instructions. Returns zero if
+// false, non-zero otherwise.
 int32_t spvOpcodeAreTypesEqual(const spv_instruction_t* pTypeInst0,
```
Comment (dmircevski) [inline source/opcode.h]: Why not rename these parameters, too?

File: source/opcode.h [hunk 7]
```diff
@@ -31,166 +31,84 @@
...
+// Determines if the given opcode results in a pointer. Returns zero if false,
+// non-zero otherwise.
 int32_t spvOpcodeIsPointer(const SpvOp opcode);
 
-/// @brief Determine if the Opcode results in a instantation of a non-void type
-///
-/// @param[in] opcode the opcode
-///
-/// @return zero if false, non-zero otherwise
+// Determines if the given opcode results in a instantation of a non-void type.
```
Comment (dmircevski) [inline source/opcode.h]: "an instantiation"

File: source/operand.h [hunk 8]
```diff
@@ -32,129 +32,96 @@
...
-/// @brief Find the operand with value in the table
-///
-/// @param[in] table to lookup
-/// @param[in] type the operand group's type
-/// @param[in] value of the operand to find
-/// @param[out] pEntry return operand table entry
-///
-/// @return result code
+// Finds the operand with value in the table. The type parameter specifies the
+// operand's group. A pointer of the operand table entry for this operand will
```
Comment (dmircevski) [inline source/operand.h]: Same comment as opcode.h:46.

File: source/opcode.h (hunk 1 above)
Comment (antiagainst) [inline source/opcode.h]: `spv_opcode_desc` is a `typedef` of `spv_opcode_desc_t*`. But yeah, may be `a handle to the table entry` is better?

Comment (dmircevski) [inline source/opcode.h]: In a public blurb, we shouldn't reveal the innards of an abstract type (like, eg, that `spv_opcode_desc` is itself a pointer).  Yeah, "a handle" or "a descriptor" would work.

File: syntax.md [hunk 9]
```diff
@@ -123,7 +123,7 @@ of `%aNaN` in the previous example is the same as the word with bits
 
 The disassembler prints infinite, NaN, and subnormal values in hexadecimal form.
 Zero and normal values are printed in decimal form with enough digits
-to preserve all significand bits.
```
Comment (dneto0) [inline syntax.md]: "significand" is correct.
https://en.wikipedia.org/wiki/Significand

File: source/opcode.h [hunk 10]
```diff
@@ -31,166 +31,84 @@
...
-
-/// @brief Get the name of the SPIR-V generator.
-///
-/// See the registry at
-/// https://www.khronos.org/registry/spir-v/api/spir-v.xml
-///
-/// @param[in] generator Khronos SPIR-V generator ID
-///
-/// @return string name
+// Gets the name of the SPIR-V generator.
```
Comment (dneto0) [inline source/opcode.h]: Returns the name of a registered SPIR-V generator as a null-terminated string.  If the generator is not known, then returns the string "Unknown".  The generator parameter should be most significant 16-bits of the generator word in the SPIR-V module header.  Values are registered at the SPIR-V XML registry: https://www.khronos.org/registry/spir-v/api/spir-v.xml

File: source/ext_inst.h [hunk 11]
```diff
@@ -30,34 +30,20 @@
 #include "libspirv/libspirv.h"
 #include "table.h"
 
-/// @brief Get the type from the extended instruction library string
-///
-/// @param name of the library
-///
-/// @return type of the extended instruction library
+// Gets the type of the extended instruction set with name.
```
Comment (dneto0) [inline source/ext_inst.h]: I'd say "with the specified name"

File: source/opcode.h (hunk 4 above)
Comment (dneto0) [inline source/opcode.h]: "instructions" -> "instruction's"

File: source/opcode.h (hunk 1 above)
Comment (antiagainst) [inline source/opcode.h]: Right. Thanks for reminding me again! :D

File: source/opcode.h (hunk 2 above)
Comment (antiagainst) [inline source/opcode.h]: Done.

File: source/opcode.h (hunk 3 above)
Comment (antiagainst) [inline source/opcode.h]: Done.

File: source/opcode.h (hunk 4 above)
Comment (antiagainst) [inline source/opcode.h]: Done.

File: source/opcode.h (hunk 5 above)
Comment (antiagainst) [inline source/opcode.h]: Done.

File: source/opcode.h (hunk 6 above)
Comment (antiagainst) [inline source/opcode.h]: Missed. :( Done.

File: source/opcode.h (hunk 7 above)
Comment (antiagainst) [inline source/opcode.h]: Done.

File: source/operand.h (hunk 8 above)
Comment (antiagainst) [inline source/operand.h]: Done.

File: syntax.md (hunk 9 above)
Comment (antiagainst) [inline syntax.md]: TIL! They are quite creative for the name. :)

File: source/opcode.h (hunk 10 above)
Comment (antiagainst) [inline source/opcode.h]: Done.

File: source/ext_inst.h (hunk 11 above)
Comment (antiagainst) [inline source/ext_inst.h]: Done.

**Comment by dmircevski** [comment]
+2

**Comment by antiagainst** [comment]
squashed comment-addressing commits into the previous one. already +2'd.

**Pull request merged**